src/integrationTests/options.test.ts

```diff
@@ -528,14 +530,14 @@ describe('Store config UID2', () => {
       const cookie = getConfigCookie();
       expect(cookie).toBeInstanceOf(Object);
       expect(cookie).toHaveProperty('cookieDomain');
-      const storageConfig = loadConfig(productDetails);
+      const storageConfig = getConfigStorage();
```

Comment:
This should have been `getConfigStorage()` all along. `loadConfig` can return the config from local storage or the cookie, but we are testing that it is not in local storage